frontend/src/locales/en.json

```diff
@@ -1,355 +1,358 @@
+    "desktop_unsafe_alert": "Some features are disabled for your protection (e.g. chat) and you will not be able to complete a trade without them. To protect your privacy and fully enable RoboSats, use <1>Tor Browser</1> and visit the <3>onion site</3>.",
+    "phone_unsafe_alert": "You will not be able to complete a trade. Use <1>Tor Browser</1> and visit the <3>onion site</3>.",
+    "Hide": "Hide",
 
 
     "USER GENERATION PAGE - UserGenPage.js": "Landing Page and User Generation",
     "Simple and Private LN P2P Exchange": "Simple and Private LN P2P Exchange",
-    "This is your trading avatar":"This is your trading avatar",
-    "Store your token safely":"Store your token safely",
-    "A robot avatar was found, welcome back!":"A robot avatar was found, welcome back!",
-    "Copied!":"Copied!",
-    "Generate a new token":"Generate a new token",
-    "Generate Robot":"Generate Robot",
-    "You must enter a new token first":"You must enter a new token first",
-    "Make Order":"Make Order",
-    "Info":"Info",
-    "View Book":"View Book",
+    "This is your trading avatar": "This is your trading avatar",
+    "Store your token safely": "Store your token safely",
+    "A Robot avatar was found, welcome back!": "A Robot avatar was found, welcome back!",
+    "Copied!": "Copied!",
+    "Generate a new token": "Generate a new token",
+    "Generate Robot": "Generate Robot",
+    "You must enter a new token first": "You must enter a new token first",
+    "Make Order": "Make Order",
+    "Info": "Info",
+    "View Book": "View Book",
 
 
     "MAKER PAGE - MakerPage.js": "This is the page where users can create new orders",
-    "Order":"Order",
-    "Customize":"Customize",
-    "Buy or Sell Bitcoin?":"Buy or Sell Bitcoin?",
-    "Buy":"Buy",
-    "Sell":"Sell",
-    "Amount":"Amount",
-    "Amount of fiat to exchange for bitcoin":"Amount of fiat to exchange for bitcoin",
-    "Invalid":"Invalid",
-    "Enter your preferred fiat payment methods. Fast methods are highly recommended.":"Enter your preferred fiat payment methods. Fast methods are highly recommended.",
-    "Must be shorter than 65 characters":"Must be shorter than 65 characters",
-    "Swap Destination(s)":"Swap Destination(s)",
-    "Fiat Payment Method(s)":"Fiat Payment Method(s)",
-    "You can add any method":"You can add any method",
-    "Add New":"Add New",
-    "Choose a Pricing Method":"Choose a Pricing Method",
-    "Relative":"Relative",
-    "Let the price move with the market":"Let the price move with the market",
-    "Premium over Market (%)":"Premium over Market (%)",
-    "Explicit":"Explicit",
-    "Set a fix amount of satoshis":"Set a fix amount of satoshis",
-    "Satoshis":"Satoshis",
-    "Let the taker chose an amount within the range":"Let the taker choose an amount within the range",
-    "Enable Amount Range":"Enable Amount Range",
+    "Order": "Order",
+    "Customize": "Customize",
+    "Buy or sell bitcoin?": "Buy or sell bitcoin?",
```

Comment:
The style of the maker page helper-texts is to capitalize all words of a sentence (See line 35, 38, 46... etc). For consistency either all helper texts are capitalized or none is. I do not have a preference.

Reply:
My *personal view is to capitalize the minimum elements except some (Lightning, Sats, Bitcoin as network...) as it's easiest to read.

Reply:
Sounds good to me! Then we can remove the capitalization of every word in the helper texts of Maker Page (es.json is also capitalized in the same sentences)